test/distrib/cpp/run_distrib_test_cmake_arm_cross.sh

```diff
@@ -20,6 +20,9 @@ cd "$(dirname "$0")/../../.."
 # Install openssl (to use instead of boringssl)
 apt-get update && apt-get install -y libssl-dev
 
+# Install arm cross-compiler
+apt-get update && apt-get install -y g++-6-arm-linux-gnueabihf
```

Comment:
another question is which arch architecture we should target.
should we perhaps test aarch64 instead of armhf? (does testing one of them give us any benefits over the other?)

Reply:
Yeah aarch64 would be better. Let me change it.